samples/filters/basic-auth-filter/pom.xml

```diff
@@ -0,0 +1,52 @@
+<!--
+  ~ Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example.tests</groupId>
+    <artifactId>basic-auth-filter</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.choreo.connect</groupId>
+            <artifactId>org.wso2.choreo.connect.enforcer.commons</artifactId>
+            <version>1.1.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.choreo.connect</groupId>
+            <artifactId>org.wso2.choreo.connect.enforcer</artifactId>
+            <version>1.0.0</version>
+        </dependency>
```

Comment:
Do we need this dependency?
If it is used to get the constant `APIConstants.MessageFormat.STATUS_CODE`, we can redefine that constants here and remove this dependency.

Reply:
No we don't. Corrected